data/models/src/commonMain/kotlin/com/tunjid/heron/data/core/models/FeedGenerator.kt

```diff
@@ -49,5 +49,6 @@ data class FeedGenerator(
         Pinned,
         Saved,
         None,
+        MoreOptions,
```

Comment:
`MoreOptions` is not really a status of the Feed. The more options button should be its own thing, completely independent of this code.